scissors/scissors_game/scissors.c

```diff
@@ -17,3 +24,8 @@ int char_to_element_index(char c)
 	}
 	return ans;
 }
+
+int play_scissors(void)
+{
+	return (rand()%3)+1;
+}
```

Comment:
Definitions order in source file and declarations order in header should be aligned.